yaml/compiler/script_win.go

```diff
@@ -35,18 +35,18 @@ func setupScriptWin(spec *engine.Spec, dst *engine.Step, src *yaml.Container) {
 // buildScriptWin is a helper script this is added to the build
 // to prepare the environment and execute the build commands.
 const buildScriptWin = `
-$ErrorActionPreference = 'Stop';
-&cmd /c "mkdir c:\root";
 if ($Env:CI_NETRC_MACHINE) {
-$netrc=[string]::Format("{0}\_netrc",$Env:HOME);
-"machine $Env:CI_NETRC_MACHINE" >> $netrc;
-"login $Env:CI_NETRC_USERNAME" >> $netrc;
-"password $Env:CI_NETRC_PASSWORD" >> $netrc;
-};
-[Environment]::SetEnvironmentVariable("CI_NETRC_PASSWORD",$null);
-[Environment]::SetEnvironmentVariable("CI_SCRIPT",$null);
-[Environment]::SetEnvironmentVariable("DRONE_NETRC_USERNAME",$null);
-[Environment]::SetEnvironmentVariable("DRONE_NETRC_PASSWORD",$null);
+@"
+machine $Env:CI_NETRC_MACHINE
+login $Env:CI_NETRC_USERNAME
+password $Env:CI_NETRC_PASSWORD
+"@ > (Join-Path $Env:USERPROFILE '_netrc');
```

Comment:
This will overwrite the file completely. Not sure if this is the correct behavior or >> which would append.